openstreetmap-carto.lua

```diff
@@ -227,6 +227,9 @@ local roads_info = {
         construction    = {z = 400, roads = false},
         platform        = {z = 90,  roads = false},
     },
+    roller_coaster = {
+        track           = {z = 300, roads = true},
```

Comment:
Don't add rollar_coaster to the roads table, and it doesn't need a z_order because it doesn't participate in the roads layering